datastore/shared/postgresql_backend/pg_connection_handler.py

```diff
@@ -162,16 +162,22 @@ def get_connection_context(self):
     def to_json(self, data):
         return Json(data)
 
-    def execute(self, query, arguments, sql_parameters=[]):
+    def execute(self, query, arguments, sql_parameters=[], use_execute_values=False):
         prepared_query = self.prepare_query(query, sql_parameters)
         with self.get_current_connection().cursor() as cursor:
-            cursor.execute(prepared_query, arguments)
+            if use_execute_values:
+                execute_values(cursor, prepared_query, arguments)
```

Comment:
You are right, somehow I thought that `None` was the default. I would just set this value "high enough", something like 1.000.000? Assuming 100 Bytes per record row, this would total to 100.000.000 Bytes = 100 MB for a single request, which should still pose absolutely no problem memory-wise and should be large enough for us to not care about it.

Reply:
Tested it a bit and it does not seem to make a measurable difference. Leaving it in anyway